cli/Cargo.toml

```diff
@@ -129,6 +129,7 @@ rand = { workspace = true, features = ["small_rng"] }
 regex.workspace = true
 reqwest.workspace = true
 ring.workspace = true
+runtimelib = "=0.9.0"
```

Comment:
Can we alias this as `jupyter_runtime`. I'm gonna have huge question marks in front of my eyes when I see it in 6 months :)